TestBenches/MatchCalculator_test.cpp

```diff
@@ -75,7 +75,7 @@ int main() {
 
   // loop over events
   for (int ievt = 0; ievt < nevents; ++ievt) {
-    //cout << "Event: " << dec << ievt << endl;
+    cout << "Event: " << dec << ievt << endl;
```

Comment:
probably should wrap this into an `ifdef __SYNTHESIS__`

Reply:
I forgot to comment that line out again after testing. It is now commented.